plugins/database/mysql/mysql.go

```diff
@@ -26,44 +26,40 @@ const (
 	`
 
 	mySQLTypeName = "mysql"
-)
 
-var (
-	MetadataLen       int = 10
-	LegacyMetadataLen int = 4
-	UsernameLen       int = 32
-	LegacyUsernameLen int = 16
+	DefaultUserNameTemplate       = `{{ printf "v-%s-%s-%s-%s" (.DisplayName | truncate 10) (.RoleName | truncate 10) (random 20) (unix_time) | truncate 32 }}`
+	DefaultLegacyUserNameTemplate = `{{ printf "v-%s-%s-%s" (.RoleName | truncate 4) (random 20) (unix_time) | truncate 16 }}`
```

Comment:
The behavior here looks to be the same as what we had, but now I'm wondering if the `(unix_time)` part ever got printed since `(random 20)` already takes 20 chars, which is well over the 16 char limit on this legacy template.

Reply:
Nope. Never did. Thanks MySQL for having ridiculous username restrictions. I just made it match the existing behavior rather than try to simplify it.

Reply:
I'd be okay to remove the `(unix_time)` bit since someone glancing at this might think that there would be a timestamp in there even though it's never the case. The context of leaving this because we had it like so will be lost over time.

Reply:
Removed